src/components/share/ShareDirective.js

```diff
@@ -47,6 +47,19 @@ goog.require('ga_permalink');
         }
         // Use clipboard API to copy URL in OS clipboard
         element.on('click', function() {
+
+            // the copy link for the iframe-preview is refreshed
+            // according to the preview changes
+            var previewFrame;
+            [previewFrame] = angular.element(document).find('iframe');
```

Comment:
Yes, in that case, we should use the existing id "gaEmbedModal" and go from there. But that's not ideal...

@mariavkourouni Can you adapt?